tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -432,9 +432,15 @@ describe(__filename, () => {
   });
 
   it('disables and enables form buttons when modification status changes', () => {
-    const renderAndCheckButtons = (shouldBeDisabled) => {
-      const root = render();
+    const name = 'Collection name';
 
+    const collection = createInternalCollection({
+      detail: createFakeCollectionDetail({ name }),
+    });
+
+    let root = render({ collection });
+
+    const buttonsAreDisabled = (shouldBeDisabled) => {
```

Comment:
I think it makes sense to have such a helper but I would pass `root` to be more explicit and maybe find a more suitable name to reflect the expectations. Could be `checkButtonsState({ root, disabled: true/false });`

Would that make sense?